scripts/utils/esbuild.ts

```diff
@@ -41,7 +42,7 @@ export async function build(options: ESBuildOptions) {
         define: Object.fromEntries(
           Object.entries(environment).map(([key, value]) => [`process.env.${key}`, JSON.stringify(value)]),
         ),
-        plugins: [...globalsPlugin(options.globals || {})],
+        plugins: [...globalsPlugin(options.globals || {}), NodeModulesPolyfillPlugin()],
```

Comment:
What are we using this for?

Reply:
To fix build errors from the wallet-connect dependency

<img width="1312" alt="Screenshot 2023-03-22 at 9 25 44 AM" src="https://user-images.githubusercontent.com/34493689/226918492-fc52a678-d1f1-452f-91cc-cdba24e49b96.png">